### DISCUSSION TITLE
Is it possible to display the PDFViewer an existing activity?

### DISCUSSION THREAD

**Discussion by angelenejoshna** [afreakyelf/Pdf-Viewer#34] [Q&A]
Hello!
I am keen on displaying a PDF file within a container of my already existing activity's layout (instead of launching a new activity). Wondering if this would be possible to do with your library? Thanks! :)

**Comment by afreakyelf** [comment]
Hi, Thank you for using this library. Above issue has been fixed in latest version.
https://github.com/afreakyelf/Pdf-Viewer/releases/tag/v2.0.1
